### ISSUE TITLE
a11y: Color Contrast - Import YAML

### ISSUE THREAD

**Issue by rak-phillip** [rancher/dashboard#13734]
Insufficient color contrast for standard text

## Actual result

The standard text with background/foreground color combinations having an insufficient contrast ratio making it difficult for people with low vision and color blindness to read standard text elements.

## Instances

"Foreground Text Color: #3D98D3
Background Text Color: #FFFFFF
Ratio: 3.16:1 
Main Content
- Read from file and Cancel buttons (on default state) 

Foreground Text Color: #FFFFFF
Background Text Color: #3D98D3
Ratio: 3.16:1 
Main Content
- Import button (on default state)

Foreground Text Color: #F4BF75
Background Text Color: #F4F5FA
Ratio: 1.53:1 
Main Content
- local
- https://vpat2.ui.rancher.space/k8s/clusters/local
- LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCk1JSUJ2RENDQ\
- kubeconfig-user-hbgtx9p2vr:glkpc8kldtz8l4b9zr7f9drtg79pzn6nl5bjzhlvsk7xmc8sgt9cks

Foreground Text Color: #B0B0B0
Background Text Color: #F4F5FA
Ratio: 1.99:1 
Main Content
- 1,2,3,4,5 and so on. 

Foreground Text Color: #F64747
Background Text Color: ##FEE4E4
Ratio: 2.94:1
- json: cannot unmarshal string into Go value of type map[string]interface {}, json: cannot unmarshal string into Go value of type map[string]interface {} and so on (which gets available on activating ""Import"" button)"

"Foreground color: #B0B0B0 
Background color: #F4F5FA
Contrast ratio: 1.99:1 
Main Content 
- Down arrow isInteractive image"

## Expected results

The contrast ratio between foreground text and background for the standard text should be sufficient.

**Comment by rak-phillip** [comment]
Fails VPAT:

> "Actual Result: The standard text with background/foreground color combinations having an insufficient contrast ratio making it difficult for people with low vision and color blindness to read standard text elements.
> 
> Instances:
> Foreground Text Color: #2F68DF
> Background Text Color: #EDF2FC
> Ratio: 4.4:1
> Main Content
> - Read from file and Cancel buttons (on hover and focus)
> 
> Foreground Text Color: #F4BF75
> Background Text Color: #F4F5FA
> Ratio: 1.53:1 
> Main Content
> - local
> - https://vpat2.ui.rancher.space/k8s/clusters/local
> -LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCk1JS UJ2RENDQ\
> - kubeconfig-user-hbgtx9p2vr:glkpc8kldtz8l4b9zr7f9drtg79pzn6nl5bjzhlvsk7xmc8sgt9cks
> 
> Foreground Text Color: #B0B0B0
> Background Text Color: #F4F5FA
> Ratio: 1.99:1 
> Main Content
> - 1,2,3,4,5 and so on. 
> 
> Foreground Text Color: #AA759F
> Background Text Color: #F4F5FA
> Ratio: 3.3:1
> - <html lang=""en"" style=""--white-space
> 
> 
> Recommendation: On all pages, ensure that each foreground/background color combination presents a contrast ratio of at least 4.5:1 for standard text."

> "The informative images present in the modal dialog has an insufficient color contrast ratio with their background
making it difficult for low vision users to discern meaningful graphics that have insufficient contrast."

Issue 126
Issue 127
